server/index.js

```diff
@@ -1,3 +1,3 @@
 require('babel-register');
-require('app-module-path').addPath(require('path').join(__dirname, '../server'));
+require('app-module-path').addPath(__dirname, '../server');
```

Comment:
@tracker1 , addPath requires only one param, doesn't it?
see [app-module-path addPath](https://github.com/patrick-steele-idem/app-module-path-node/blob/master/lib/index.js)